docs/api/scheduler.md

```diff
@@ -528,13 +528,20 @@ partitions:
         submitacl: '*'
         properties:
           application.sort.policy: stateaware
+checksum: BAB3D76402827EABE62FA7E4C6BCF4D8DD9552834561B6B660EF37FED9299791
 ```
+**Note:** the checksum is the base configuration checksum on top of what we want to make the changes.
+If the provided checksum in the request differs from the base version, the configuration update will fail.
+This is to avoid applying any unintentional changes to the running version.
+Subsequently, the client needs to retrieve the latest version from the scheduler and send another update with the up-to-date checksum.
 
 ### Failure response
 
-The configuration update can fail due to invalid configuration 
+The configuration update can fail due to different reasons such as:
+- invalid configuration,
+- outdated base checksum.
```

Comment:
It is not an outdated checksum it is the incorrect checksum. We have no idea if it is outdated or just some random string.